models/Post.php

```diff
@@ -242,21 +242,16 @@ public function scopeListFrontEnd($query, $options)
         /*
          * Sorting
          */
-        if (!is_array($sort)) {
-            $sort = [$sort];
-        }
-
-        foreach ($sort as $_sort) {
-
-            if (in_array($_sort, array_keys(self::$allowedSortingOptions))) {
-                $parts = explode(' ', $_sort);
-                if (count($parts) < 2) {
-                    array_push($parts, 'desc');
-                }
-                list($sortField, $sortDirection) = $parts;
-                if ($sortField == 'random') {
-                    $sortField = Db::raw('RAND()');
+        if (in_array($sort, array_keys(self::$allowedSortingOptions))) {
```

Comment:
wouldn't `static::$allowedSortingOptions` be better here than `self::` to allow for people overriding the model?

Reply:
Done